src/redux/actions/index.js

```diff
@@ -0,0 +1,24 @@
+export const isLoading = (loading) => ({
+  type: "IS_LOADING",
+  loading
+});
+
+export const hasErrored = (message) => ({
+  type: "HAS_ERRORED",
+  message
+});
+
+export const pushPalette = (palette) => ({
+  type: "PUSH_PALETTE",
+  palette
+});
+
+export const createProject = (project) => ({
+  type: "CREATE_PALETTE",
+  project
+});
```

Comment:
Did you mean this to be create palette or create project? We probably want one of each right?